inventory/sample.aws.example.com.d/inventory/group_vars/OSEv3.yml

```diff
@@ -9,10 +9,23 @@ ansible_become: true
 # Node Labels to use with the CASL tools
 openshift_node_labels: "{{ ec2_tag_node_labels }}"
 
+# CNS relative vars
```

Comment:
As CNS is defaulted to `false` (which is correct), please comment these out to keep them as an example, but doesn't give the impression that CNS is enabled

Reply:
Done